servers/visual/rasterizer.cpp

```diff
@@ -33,6 +33,10 @@
 #include "core/os/os.h"
 #include "core/print_string.h"
 
+#if defined(DEBUG_ENABLED) && defined(TOOLS_ENABLED)
```

Comment:
`TOOLS_ENABLED` assumes `DEBUG_ENABLED` so you don't need to check the latter.

Reply:
That being said if you prefer to have both to make the intent extremely clear, that's fine too.

Reply:
Yes I remember this coming up before. I'm slightly in favour of keeping both as you say to make the intent clear, because to a reader it is not obvious that `TOOLS_ENABLED` implies `DEBUG_ENABLED` (and this could conceivably change in future).